src/modules/ui.js

```diff
@@ -69,7 +75,23 @@ export const getDefaultSorting = () => ({
     direction: 'default',
 })
 
-export const ACCESSORY_PANEL_TAB_INPUT = 'INPUT'
-export const ACCESSORY_PANEL_TAB_PROGRAM = 'PROGRAM'
-export const ACCESSORY_PANEL_TAB_YOUR = 'YOUR'
-export const ACCESSORY_PANEL_TAB_TRACKED_ENTITY = 'TRACKED_ENTITY'
+export const getUserSidebarWidth = () => {
```

Comment:
In dashboard the code that deals with `localStorage` is in a separate file called `modules/localStorage.js`, but we have more keys there used in different places.

Reply:
Good point. I wasn't quite sure what to do with this when I wrote the code. I've implemented it in a separate file now, see [3023350](https://github.com/dhis2/line-listing-app/pull/534/commits/3023350ea8368d2b5c16b2938626701408cef633)